tests/test_modes.py

```diff
@@ -108,3 +115,61 @@ def test_should_list_branch_changed_files_as_affected_tests(self):
 
         assert files == expected_files
         assert folders == expected_folders
+
+    def test_should_list_changed_files(self, testdir):
+        # initialize a new git repo in testdir/gitroot
+        gitroot = testdir.mkdir("gitroot")
+        gitroot.chdir()
+        try:
+            assert testdir.run("git", "init").ret == 0
+        except FileNotFoundError:
+            pytest.skip("required executable 'git' not found")
+
+        # provide git-commit a valid committer and author
+        if "EMAIL" not in os.environ:
```

Comment:
Hi @relud. I asked you on previous review but I'm afraid you didn't answer it: why do we need this? I just tested and the tests are passing without it and also I rather not set `EMAIL` as environment variable every time a test runs. Unless you have a good reason, we're good to remove this if :)

Reply:
ah, my bad, i apparently failed to click `comment` >_<

the tests were failing for me without this in CI because git couldn't detect a valid user email. I will remove it.